lvis/eval.py

```diff
@@ -521,10 +529,17 @@ def run(self):
         self.summarize()
 
     def print_results(self):
-        template = " {:<18} {} @[ IoU={:<9} | area={:>6s} | maxDets={:>3d} catIds={:>3s}] = {:0.3f}"
+        template = " {:<18} {} @[ IoU={:<9} | area={:>6s} | maxDets={} catIds={:>3s}] = {:0.3f}"
 
         for key, value in self.results.items():
-            max_dets = self.params.max_dets
+            max_dets_im = self.params.max_dets
+            max_dets_cat = self.params.max_dets_per_cat
+            if max_dets_cat < 0:
+                max_dets_str = str(max_dets_im)
```

Comment:
Should we use `"{:>3d}".format(max_dets_im)` to preserve the previous formatting?